WordPress/Classes/ViewRelated/Jetpack/Branding/Fullscreen Overlay/JetpackPluginOverlayViewModel.swift

```diff
@@ -0,0 +1,163 @@
+import Foundation
+
+class JetpackPluginOverlayViewModel: JetpackFullscreenOverlayViewModel {
+    private enum Constants {
+        static let lottieLTRFileName = "JetpackInstallPluginLogoAnimation_ltr"
+        static let lottieRTLFileName = "JetpackInstallPluginLogoAnimation_rtl"
+    }
+
+    let title: String = Strings.title
+    let subtitle: NSAttributedString
+    let animationLtr: String = Constants.lottieLTRFileName
+    let animationRtl: String = Constants.lottieRTLFileName
+    let footnote: String? = nil
+    let actionInfoText: NSAttributedString? = JetpackPluginOverlayViewModel.footnote()
+    let learnMoreButtonURL: String? = nil
```

Comment:
I'm not sure which button this is. Is this for the terms & conditions URL? 🤔 Seeing that this is still in progress, I guess this would be clearer in your next PRs.